<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcClass</class>
 <widget class="QMainWindow" name="CalcClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calc</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Rez_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>34</y>
      <width>360</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: none;
}

background-color: white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_invert">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
background-color: rgb(224, 224, 224);
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
						  stop: 0#C0C0C0, stop: 1#D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
						  stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
						  stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
						  stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_centry">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(202, 255, 255);
border: none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0, y1:0, z2:0, y2:1,
						  stop: 0#A2B5CD, stop: 1#BCD2EE);
}</string>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sgnch">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
background-color: rgb(224, 224, 224);
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
						  stop: 0#C0C0C0, stop: 1#D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>±</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
background-color: rgb(224, 224, 224);
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
						  stop: 0#C0C0C0, stop: 1#D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(202, 255, 255);
border: none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0, y1:0, z2:0, y2:1,
						  stop: 0#A2B5CD, stop: 1#BCD2EE);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
						  stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
						  stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
						  stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
						  stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_decimal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
						  stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_result">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>390</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(202, 255, 255);
border: none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0, y1:0, z2:0, y2:1,
						  stop: 0#A2B5CD, stop: 1#BCD2EE);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addition">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(202, 255, 255);
border: none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0, y1:0, z2:0, y2:1,
						  stop: 0#A2B5CD, stop: 1#BCD2EE);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
						  stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
						  stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_substraction">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(202, 255, 255);
border: none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0, y1:0, z2:0, y2:1,
						  stop: 0#A2B5CD, stop: 1#BCD2EE);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
						  stop: 0#dadbde, stop: 1#f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiplication">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(202, 255, 255);
border: none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0, y1:0, z2:0, y2:1,
						  stop: 0#A2B5CD, stop: 1#BCD2EE);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqr">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:none;
background-color: rgb(224, 224, 224);
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
						  stop: 0#C0C0C0, stop: 1#D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>x^2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_division">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(202, 255, 255);
border: none;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0, y1:0, z2:0, y2:1,
						  stop: 0#A2B5CD, stop: 1#BCD2EE);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>360</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	qproperty-alignment: 'AlignVCenter | AlignLeft';
	border: none;
}

background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Calc.qrc"/>
 </resources>
 <connections/>
</ui>
